build-tools/packages/build-tools/src/repoPolicyCheck/handlers/fluidBuildTasks.ts

```diff
@@ -146,9 +146,12 @@ function getDefaultTscTaskDependencies(root: string, json: PackageJson) {
 
 function findTscScript(json: PackageJson, project: string) {
 	if (project === "./tsconfig.json") {
-		return findScript(json, "tsc");
+		return findScript(json, "tsc") || findScript(json, "tsc-multi --config tsc-multi.cjs.json");
```

Comment:
Do we need the new script check to be exact including the argument?

Reply:
Thanks, your comment made me realize I had a bug. Fixed.